cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesAliasNamingStrategyFixOnlyIidmIdentifiableInvalidIds.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Identifiable;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at rte-france.com>
+ */
+public class CgmesAliasNamingStrategyFixOnlyIidmIdentifiableInvalidIds extends CgmesAliasNamingStrategyFixInvalidIds {
```

Comment:
Rather than extending the other class, I would advise to use a common abstract parent class. Otherwise, we might not control evolutions (if we forget to override something for example)...

Reply:
I've followed your advice and created an abstract class that is a common parent for the two implementations. I've added an abstract method `getName`, that also will be useful when we refactor these classes with service loaders.